acceptancetests/jujupy/client.py

```diff
@@ -1142,7 +1142,7 @@ def get_juju_output(self, command, *args, **kwargs):
         """
         model = self._cmd_model(kwargs.get('include_e', True),
                                 kwargs.get('controller', False))
-        return self.get_raw_juju_output(self, command, model, *args, **kwargs)
+        return self.get_raw_juju_output(command, model, *args, **kwargs)
 
     def get_raw_juju_output(self, command, model, *args, **kwargs):
```

Comment:
should we put a default value for model in the def of get_raw_juju_output?

Reply:
I don't think we need to here, but we can re-evaluate at a later time.